http_connector.py

```diff
@@ -367,62 +425,76 @@ def _make_http_call(self, action_result, endpoint='', method='get', headers=None
                 url=url,
                 auth=auth,
                 params=params,
-                data=UnicodeDammit(data).unicode_markup.encode('utf-8') if isinstance(data, str) else data,
+                data=UnicodeDammit(data).unicode_markup.encode("utf-8")
+                if isinstance(data, str)
+                else data,
                 verify=verify,
                 headers=headers,
                 files=files,
-                timeout=self._timeout)
+                timeout=self._timeout,
+            )
 
         except Exception as e:
             error_message = self._get_error_message_from_exception(e)
-            return action_result.set_status(phantom.APP_ERROR, "Error Connecting to server. Details: {0}".format(error_message)), None
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                "Error Connecting to server. Details: {0}".format(error_message),
+            ), None
 
         # fetch new token if old one has expired
         if access_token and r.status_code == 401 and self.access_token_retry:
             self.save_progress("Got error: {}".format(r.status_code))
-            self._state.pop('access_token')
+            self._state.pop("access_token")
             self.access_token_retry = False  # make it to false to avoid getting access token after one time (prevents recursive loop)
-            return self._make_http_call(action_result, endpoint, method, headers, verify, data)
+            return self._make_http_call(
+                action_result,
+                endpoint=endpoint,
+                method=method,
+                headers=headers,
+                verify=verify,
+                data=data,
+            )
```

Comment:
Since this is a recursive call, should probably add all parameters of `_make_http_call` here (`params`, `files`, and `use_default_endpoint`):

```py
            return self._make_http_call(
                action_result,
                endpoint=endpoint,
                method=method,
                headers=headers,
                params=params,
                verify=verify,
                data=data,
                files=files,
                use_default_endpoint=use_default_endpoint,
            )
```